content/guides/hosting/hosting-options/self-managed/kubernetes-operator/_index.md

```diff
@@ -465,28 +465,67 @@ global:
     secretKey: HDKYe4Q...JAp1YyjysnX
 ```
 
+The `kmsKey` must be `null`.
+
+To reference `accessKey` and `secretKey` from a secret:
+```yaml
+global:
+  bucket:
+    # Example values, replace with your own
+    provider: s3
+    name: storage.example.com
+    kmsKey: null
+    path: wandb
+    region: default
+    secret:
+      secretName: wandb-bucket
+      accessKeyName: ACCESS_KEY
+      secretKeyName: SECRET_KEY
+```
+
 ### MySQL
 
 ```yaml
 global:
    mysql:
      # Example values, replace with your own
+     host: db.example.com
+     port: 3306
      database: wandb_local
-     host: 10.218.0.2
-     name: wandb_local
-     password: 8wtX6cJH...ZcUarK4zZGjpV
+     user: wandb
+     password: 8wtX6cJH...ZcUarK4zZGjpV 
+```
+
+To reference the `password` from a secret:
+```yaml
+global:
+   mysql:
+     # Example values, replace with your own
+     host: db.example.com
      port: 3306
+     database: wandb_local
      user: wandb
+     passwordSecret:
+       name: mysql-wandb-credentials
```

Comment:
Same here but just to standardize the secret names, I suggest `database-secret`
Using `mysql-secret` might clash with some MySQL from some other MySQL deployment.